### PR TITLE
LICENCE & Visitors tag added

### PR DESCRIPTION
**Pull request by kaiwalyakoparkar**
Licence & Visitors tag has been added to the Readme file. 
- ## Pros
-Any visitor can see the popularity of the repo directly.
-This will attract more contributors to it.
-Licence is in the seperate file which is not usually read by contributors
-Adding this will increase the chances of the contributors to read it and this will reduce the inaccuracy of PRs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,4 +52,36 @@ _For all three options, fill in all fields that apply._
 
 ## License
 
-WhatsApp Stickers is BSD licensed, as found in the [`LICENSE`](https://github.com/WhatsApp/stickers/blob/master/LICENSE) file.
+WhatsApp Stickers is BSD licensed as below,
```
Comment (Zandor300) [inline README.md:55]: I don't see how this is useful.

File: README.md [hunk 2]
```diff
@@ -17,7 +19,7 @@ For iOS developers please read the [IMPORTANT NOTICE ABOUT iOS APPS](#please-rea
 
 To get started, review the `README` files in the [Android](https://github.com/WhatsApp/stickers/tree/master/Android) or [iOS](https://github.com/WhatsApp/stickers/tree/master/iOS) folders, and refer to the [FAQ](https://faq.whatsapp.com/general/26000226).
 
-## PLEASE READ: IMPORTANT NOTICE ABOUT iOS APPS
+## PLEASE READ: IMPORTANT NOTICE REGARDING iOS APPS
```
Comment (Zandor300) [inline README.md:22]: This is fine.

File: README.md [hunk 3]
```diff
@@ -3,6 +3,8 @@
 ![Works with Android](https://img.shields.io/badge/Works_with-Android-green?style=flat-square)
 ![Works with iOS](https://img.shields.io/badge/Works_with-iOS-blue?style=flat-square)
 ![GitHub issues](https://img.shields.io/github/issues/whatsapp/stickers?style=flat-square)
+![Visitors](https://visitor-badge.glitch.me/badge?page_id=stickers.stickers)
```
Comment (Zandor300) [inline README.md:6]: I would change this to the following to match actual repository url:
```suggestion
![Visitors](https://visitor-badge.glitch.me/badge?page_id=WhatsApp.stickers)
```

**Review by 80014917 — COMMENTED** [review]
Sorey

**Pull request closed without merging**